netdisco/discoverables/yeelight.py

```diff
@@ -30,11 +31,12 @@ def info_from_entry(self, entry):
         elif entry.name.startswith("yeelink-light-ceiling2_"):
             device_type = "ceiling2"
         else:
-            logging.warning("Unknown miio device found: %s", entry)
+            device_type = \
+                entry.name.replace(DEVICE_NAME_PREFIX, '').rsplit('_', 1)[0]
```

Comment:
Why wouldn't we always do this? It seems unnecessary for us to make up our own device types instead of using the ones provided by Yeelink.

Reply:
It will only be a breaking change if not resolved in Home Assistant. I think that it is weird that a discovery library is introducing their own names for types. We should follow the vendor types.